datastore/prometheus.go

```diff
@@ -188,18 +202,43 @@ func (p *PrometheusExporter) handlePacket(pkt Packet) {
 			fromPod.(PodEvent).Labels[appNamespaceLabel] == toPod.(PodEvent).Labels[appNamespaceLabel] &&
 			fromPod.(PodEvent).Labels[appLabel] == toPod.(PodEvent).Labels[appLabel] {
 
-			labels["fromPod"] = fromPod.(PodEvent).Name
-			labels["fromAcornApp"] = fromPod.(PodEvent).Labels[appLabel]
-			labels["fromAcornAppNamespace"] = fromPod.(PodEvent).Labels[appNamespaceLabel]
-			labels["fromAcornContainer"] = fromPod.(PodEvent).Labels[containerLabel]
-
-			labels["toPod"] = toPod.(PodEvent).Name
-			labels["toAcornApp"] = toPod.(PodEvent).Labels[appLabel]
-			labels["toAcornAppNamespace"] = toPod.(PodEvent).Labels[appNamespaceLabel]
-			labels["toAcornContainer"] = toPod.(PodEvent).Labels[containerLabel]
+			labels := prometheus.Labels{
+				"toPort":                strconv.Itoa(int(pkt.ToPort)),
+				"fromPod":               fromPod.(PodEvent).Name,
+				"fromAcornApp":          fromPod.(PodEvent).Labels[appLabel],
+				"fromAcornAppNamespace": fromPod.(PodEvent).Labels[appNamespaceLabel],
+				"fromAcornContainer":    fromPod.(PodEvent).Labels[containerLabel],
+				"toPod":                 toPod.(PodEvent).Name,
+				"toAcornApp":            toPod.(PodEvent).Labels[appLabel],
+				"toAcornAppNamespace":   toPod.(PodEvent).Labels[appNamespaceLabel],
+				"toAcornContainer":      toPod.(PodEvent).Labels[containerLabel],
+			}
 
 			p.throughputCounter.With(setEmptyPrometheusLabels(labels, throughputCounterLabels)).Add(float64(pkt.Size))
 		}
+	} else if pkt.IsIngress && pkt.FromType == PodSource && pkt.ToType == OutsideDest && !strings.HasPrefix(pkt.ToIP, "10.") {
```

Comment:
I assume that `pkt.ToType == OutsideDest` is not good enough to determine whether it's ingress or egress traffic? :thinking: 

Also, you may want to add a similar tiny note like on the struct explaining that `pkt.Ingress` means egress metric - confused me even though I read it above :sweat_smile: